pkg/controller/services.go

```diff
@@ -300,13 +300,139 @@ func deleteDevicesFromList(delDevices apicapi.ApicSlice, devices apicapi.ApicSli
 	}
 	return newDevices
 }
+func (cont *AciController) getNodeName(nodeName string) string {
+	nodeList := cont.nodeIndexer.List()
+	for _, nodeItem := range nodeList {
+		node := nodeItem.(*v1.Node)
+		if nodeName == node.ObjectMeta.Name {
+			ip := getNodeIP(node, v1.NodeInternalIP)
+			return ip
+		}
+	}
+	return ""
+}
+
+func (cont *AciController) getfvRsCEpToPathEptDn(node string) (string, error) {
+	ipFilter := fmt.Sprintf("query-target-filter=and(eq(fvCEp.ip,\"%s\"))", node)
+	args := []string{
+		ipFilter,
+		"rsp-subtree-class=fvRsCEpToPathEp",
+		"rsp-subtree=children",
+	}
+	url := fmt.Sprintf("/api/node/class/fvCEp.json?%s", strings.Join(args, "&"))
+	apicresp, err := cont.apicConn.GetApicResponse(url)
+	if err != nil {
+		cont.log.Debug("Failed to get APIC response, err: ", err.Error())
+		return "", err
+	}
+	for _, obj := range apicresp.Imdata {
+		for _, body := range obj {
+			for _, child := range body.Children {
+				for class, cbody := range child {
+					if class == "fvRsCEpToPathEp" {
+						tDn, ok := cbody.Attributes["tDn"].(string)
+						if ok {
```

Comment:
It is possible that another cluster can have an overlapping ip. Please check here that either Dn of fvRsCEpToPathEp has the tenant name as uni/<tn-name>/xxx or the parent fvCEp has it in its Dn.